module5/intermission/index.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: Intermission Work
+---
+
+Before starting Module 5, spend a minimum of 2 hours working through these [PostgreSQL Exercises](https://pgexercises.com/).
```

Comment:
Good find with these exercises!! At first, I was unsure about how they all have the option to view the solution, but now I'm feeling that at this point if students don't want to put the time in to learn/review that's on them. I like the hint option a lot!